Team8745/src/main/java/org/firstinspires/ftc/team8745/Robot.java

```diff
@@ -0,0 +1,43 @@
+package org.firstinspires.ftc.team8745;
+
+import com.qualcomm.robotcore.hardware.ColorSensor;
+import com.qualcomm.robotcore.hardware.DcMotor;
+import com.qualcomm.robotcore.hardware.DcMotorSimple;
+import com.qualcomm.robotcore.hardware.HardwareMap;
+import com.qualcomm.robotcore.hardware.Servo;
+
+
+/**
+ * Created by rose on 10/8/17.
+ */
+
+public class Robot {
+
+    public ColorSensor colorSensor;
+    public Servo jewelKnockOff;
+    public DcMotor left_b;
```

Comment:
left_b could be anything left and back, so use camelCase and write something like leftBackMotor.